hub/instruction_handling/call/generalities/universal.tex

```diff
@@ -124,18 +124,18 @@
 	\item[\underline{Setting \mxpMod{} instruction parameters:}]
 		\If $\miscMxpFlag_{i + \callMiscRowOffset} = 1$ \Then we impose that
 		\[
-			\setMxpInstructionTypeFive {
-				anchorRow          = i,
-				relOffset          = \callMiscRowOffset,
-				instruction        = \locInst,
-				callDataOffsetHi   = \locCdoHi,
-				callDataOffsetLo   = \locCdoLo,
-				callDataSizeHi     = \locCdsHi,
-				callDataSizeLo     = \locCdsLo,
-				returnAtOffsetHi   = \locRaoHi,
-				returnAtOffsetLo   = \locRaoLo,
-				returnAtCapacityHi = \locRacHi,
-				returnAtCapacityLo = \locRacLo,
+			\setMxpInstructionCalls {
```

Comment:
This and other modifications remove the `TYPE_1/2/3/4/5` stuff from the HUB as it was removed from the MXP module.